src/components/edit/LayerEdit.js

```diff
@@ -80,7 +83,7 @@ const LayerEdit = ({ layer, addLayer, updateLayer, cancelLayer }) => {
         return null
     }
 
-    let name = layerName()[type]
+    let name = layerName()[type] || layer.type
```

Comment:
@turban do you remember the situation where this change was needed?

Reply:
I think we can remove ` || layer.type` here. I have never liked that we use `layer.layer`, but I've suggested that we change it to `layer.layerType` in the EE branch I'm working on.